src/T8code.jl

```diff
@@ -157,6 +159,84 @@ not a system-provided `t8code` installation. In this case, T8code.jl is not usab
 preferences_set_correctly() = !(_PREFERENCE_LIBT8 == "t8code_jll" &&
                                 MPIPreferences.binary == "system")
 
+# Minimal reference tracker holding active t8code related objects
+# created throughout the life time of a Julia session. t8code objects
+# should remove themselves from the tracker when they get finalized.
+if !@isdefined(T8CODE_OBJECT_TRACKER)
+    T8CODE_OBJECT_TRACKER = Dict()
```

Comment:
```suggestion
    T8CODE_OBJECT_TRACKER = Dict{UInt64, ForestWrapper}()
```
`Dict` gives `Dict{Any,Any}`.

Reply:
Let's be explicit here. Fixed.